Repository/Courses/CourseRepository.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Interface.Repositories;
+using Model.DTOs;
+using Repository.DbModels;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Repository.Courses
+{
+    public class CourseRepository : ICourseRepository
+    {
+        private readonly DbModels.DbModels context;
+
+        public CourseRepository(DbModels.DbModels context)
+        {
+            this.context = context;
+        }
+
+        public CourseDTO Add(CourseDTO course) 
+        {
+            context.Courses.Add(new DbModels.Course
```

Comment:
This should be done inside `try/ catch`, because right now you aren't validating any value that is sent and this is error prone